stable/linkerd/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: Service mesh for cloud native apps
+name: linkerd
```

Comment:
Can you add the appropriate metadata for home, source, maintainers, icon and some keywords? e.g.

```
home: https://linkerd.io/
sources:
- https://github.com/BuoyantIO/linkerd
maintainers:
- name: Vic Iglesias
  email: viglesias@google.com
```